src/Command/GenerateSchema.php

```diff
@@ -82,9 +91,13 @@ public function execute(InputInterface $input, OutputInterface $output)
     {
         $data = $this->schema->getSchema();
 
+        $this->logger->info('Starting schema dist file generation');
+
         $this->file->filePutContents(
             $this->fileList->getEnvDistConfig(),
             $this->formatter->format($data)
         );
+
+        $this->logger->info(sprintf('Dist file was successfully generated: %s', $this->fileList->getEnvDistConfig()));
```

Comment:
Logging in command classes does not have much value, Command output should be used instead

Reply:
Changed